lambda/security-groups/src/main/scala/com/gu/hq/Lambda.scala

```diff
@@ -23,9 +24,10 @@ class Lambda extends RequestHandler[ConfigEvent, Unit] with StrictLogging {
       sgConfiguration <- ConfigEventLogic.sgConfiguration(configurationItem.configuration)
       loadBalancers = AWS.describeLoadBalancers(elbClient)
       account = AWS.accountNumber(stsClient)
+      accountName = AWS.accountName(s3Client, account)
```

Comment:
As before, this is two steps.

1. Fetch the Account name config
2. Apply the logic to lookup the account name from that config

Reply:
Done.